src/sentry/options/defaults.py

```diff
@@ -269,6 +269,11 @@
     flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+register(
+    "api.prevent-numeric-slugs",
+    default=0,
+)
```

Comment:
should we be specifying the flags parameter here? like `flags=DEFAULT_FLAGS` or `flag=FLAG_IMMUTABLE`? is there a reason we set it to `0` instead of `False`?

i see that we're checking if it's > 0 but would setting it to a boolean allow us to check the existence by itself?

Reply:
Only because I followed dev guide :P and it says nothing about flags or booleans 😅  https://develop.sentry.dev/options/

Let me fix that.